runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -1150,6 +1150,18 @@ J9::AheadOfTimeCompile::initializeCommonAOTRelocationHeader(TR::IteratedExternal
          }
          break;
 
+      case TR_Breakpoint:
```

Comment:
This commit deserves a non-empty message summarizing these changes :) .

Reply:
That's true of the first 3 commits, actually :) .

Sorry I left this comment hanging here! i got distracted after making this one comment and have been on vacation until today.

Reply:
Addressed in the latest force push.